### PR TITLE
Add tradeDateTime to Trade type and Execution Instruction

### PR DESCRIPTION
**Pull request by tomhealey-icma**
Reference issue,[2205](https://github.com/finos/common-domain-model/issues/2205)

### CODE REVIEW

File: documentation/source/event-model.rst [hunk 1]
```diff
@@ -63,20 +63,22 @@ The ``Trade`` data type defines the outcome of a financial transaction between p
...
+	tradeIdentifier TradeIdentifier (1..*)
+	tradeDate date (1..1)
+        [metadata id]
+	tradeDateTime zonedDateTime (0..1)
+        [metadata id]
+	tradableProduct TradableProduct (1..1)
+	party Party (0..*)
+	partyRole PartyRole (0..*)
+	executionDetails ExecutionDetails (0..1)
+	contractDetails ContractDetails (0..1)
```
Comment (chrisisla) [inline documentation/source/event-model.rst:76]: Is contracttDetails being added in this PR as well?

File: rosetta-source/src/main/rosetta/event-common-type.rosetta [hunk 2]
```diff
@@ -103,6 +103,8 @@ type ExecutionInstruction: <"Specifies instructions for execution of a transacti
 	executionDetails ExecutionDetails (1..1) <"Specifies the type and venue of execution, e.g. via voice, or electronically.">
 	tradeDate date (1..1) <"Denotes the trade/execution date.">
 	    [metadata id]
+	tradeDateTime zonedDateTime (0..1) <"Denotes the trade execution date and time with timezone.">
```
Comment (chrisisla) [inline rosetta-source/src/main/rosetta/event-common-type.rosetta:106]: As per my comment on the associated Issue I think we should have separate trade date and trade time/timezone items.

**Review by chrisisla — CHANGES_REQUESTED** [review]

**Comment by tomhealey-icma** [comment]
I agree with this approach. We'll keep tradeDate as is and add tradeTime TimeZone (0..1) which adds the other two attributes time and timezone that make up zonedDateTime.

File: documentation/source/event-model.rst (hunk 1 above)
Comment (tomhealey-icma) [reply documentation/source/event-model.rst:76]: No that is probably just the result of indenting.

**Comment by lolabeis** [comment]
Please provide release notes in .md format as a comment in the PR before it is merged, so that notes can be added to the release tag.

You can follow these guidelines:
https://cdm.docs.rosetta-technology.io/source/contribution.html#content-of-release-notes

**Pull request merged**